### PR TITLE
Add JS and CU to 2.0.1 manifest

### LINKED ISSUE #2165: [RELEASE] Release version 2.0.1
## Release OpenSearch and OpenSearch Dashboards 2.0.1

I noticed that a manifest was automatically created in [manifests/2.0.1](/opensearch-project/opensearch-build/tree/main/manifests/2.0.1). Please follow the following checklist to make a release.

<details><summary>How to use this issue</summary>
<p>

## This Release Issue

This issue captures the state of the OpenSearch release, its assignee is responsible for driving the release. Please contact them or @mention them on this issue for help. There are linked issues on components of the release where individual components can be tracked.  More details are included in the Maintainers [Release owner](https://github.com/opensearch-project/opensearch-build/blob/main/MAINTAINERS.md#release-owner) section.

## Release Steps

There are several steps to the release process, these steps are completed as the whole release and components that are behind present risk to the release.  The release owner completes the tasks in this ticket, whereas component owners resolve tasks on their ticket in their repositories.

Steps have completion dates for coordinating efforts between the components of a release; components can start as soon as they are ready far in advance of a future release.

### Component List

To aid in understanding the state of the release there is a table with status indicating each component state. This is updated based on the status of the component issues.

</p>
</details>

### Preparation

- [x] Assign this issue to a release owner.
- [x] Declare a pencils down date for new features to be merged. 
- [x] June 8th is pencils down date for feature freeze.
- [ ] ~~Update the Campaigns section to include monitoring campaigns during this release.~~
- [x] Update this issue so all `__REPLACE_RELEASE-__` placeholders have actual dates.
- [x] Document any new quality requirements or changes.
- [x] Finalize scope and feature set and update [the Public Roadmap](https://github.com/orgs/opensearch-project/projects/1). 
- [ ] ~~[Create a release issue in every component repo](https://github.com/opensearch-project/opensearch-build/blob/main/meta/README.md#create-a-release-issue) that links back to this issue, update Components section with these links.~~ (We don't create component release issue for patch release.)
- [x] Ensure the label is created in each component repo for this new version, and the next minor release. [Create a version label](https://github.com/opensearch-project/opensearch-plugins/blob/main/META.md#create-or-update-labels-in-all-plugin-repos)
- [ ] ~~Ensure that all release issues created above are assigned to an owner in the component team.~~

### CI/CD (Feature Freeze) - _Ends June 8th

- [x] Create Jenkins workflows that run daily snapshot builds for OpenSearch and OpenSearch Dashboards. 
- [x] Increment each component version to 2.0.1 and ensure working CI in component repositories.
- [x] Make pull requests to add each component to [manifests/2.0.1/opensearch-2.0.1.yml](/opensearch-project/opensearch-build/tree/main/manifests/2.0.1/opensearch-2.0.1.yml) and [manifests/2.0.1/opensearch-dashboards-2.0.1.yml](/opensearch-project/opensearch-build/tree/main/manifests/2.0.1/opensearch-dashboards-2.0.1.yml) with the corresponding checks.
- [x] OpenSearch / OpenSearch-Dashboards core and components teams finalize their features

### Code Complete - _Ends June 9th

- [x] Code Complete: Make sure that the code for this specific version of the release is ready and the branch corresponding to this release has been added to this release version manifest.
- [x] Verify pull requests to add each component to [manifests/2.0.1/opensearch-2.0.1.yml](/opensearch-project/opensearch-build/tree/main/manifests/2.0.1/opensearch-2.0.1.yml) and [manifests/2.0.1/opensearch-dashboards-2.0.1.yml](/opensearch-project/opensearch-build/tree/main/manifests/2.0.1/opensearch-dashboards-2.0.1.yml) have been merged.
- [x] Gather, review and combine the release notes from components repositories.
    - https://github.com/opensearch-project/opensearch-build/pull/2202

### Release testing - _Ends June 15th

- [x] Declare a release candidate build, and provide the instructions with the release candidates for teams on testing (__REPLACE_RELEASE-minus-8-days__).
- [x] Sanity Testing (__REPLACE_RELEASE-minus-8-days__ - __REPLACE_RELEASE-minus-6-days__): Sanity testing and fixing of critical issues found by teams. Teams test their components within the distribution, ensuring integration, backwards compatibility, and perf tests pass.
- [x] Publish all test results in the comments of this issue.

### Release - _Ends June 16th

- [x] Verify [all issues labeled `v2.0.1` in all projects](https://github.com/opensearch-project/project-meta#find-labeled-issues) have been resolved.
- [x] Complete [documentation](https://github.com/opensearch-project/documentation-website) for this release.
- [x] Author [blog post](https://github.com/opensearch-project/project-website) for this release.
- [x] Publish this release on [opensearch.org](https://opensearch.org/downloads.html).
- [x] Publish a [blog post](https://github.com/opensearch-project/project-website) - release is launched!

### Post Release

- [x] Create [release tags](https://github.com/opensearch-project/opensearch-build/blob/main/jenkins/release-tag/release-tag.jenkinsfile) for each component (Jenkins job name: release-tag-creation).
- [x] Replace refs in [manifests/2.0.1](/opensearch-project/opensearch-build/tree/main/manifests/2.0.1) with tags and remove checks.
    - https://github.com/opensearch-project/opensearch-build/pull/2204
- [x] Generate distribution release notes reviewed by PM team for opensearch-build repository.
- [x] Increment version for Helm Charts [(sample PR)](https://github.com/opensearch-project/helm-charts/pull/246) for the `2.0.1` release.
    - https://github.com/opensearch-project/helm-charts/pull/277
- [x] Increment version for Ansible Charts [(sample PR)](https://github.com/opensearch-project/ansible-playbook/pull/50) for the `2.0.1` release.
    - https://github.com/opensearch-project/ansible-playbook/pull/74
- [x] Prepare [for next patch release](https://github.com/opensearch-project/opensearch-plugins/blob/main/META.md#increment-a-version-in-every-plugin) by incrementing patch versions for each component.
    - https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1758
    - https://github.com/opensearch-project/OpenSearch/pull/3611
- [x] Lower the [frequency of builds](https://github.com/opensearch-project/opensearch-build/pull/1475) for this version of OpenSearch and/or OpenSearch Dashboards.
- [x] Update [this template](https://github.com/opensearch-project/opensearch-build/blob/main/.github/ISSUE_TEMPLATE/release_template.md) with any new or missed steps.
- [x] Create an issue for a retrospective, solicit feedback, and publish a summary.
    - https://github.com/opensearch-project/opensearch-build/issues/2208https://github.com/opensearch-project/opensearch-build/issues/2208

### Components

__Replace with links to all component tracking issues.__

| Component | On track | Notes |
| --------- | -------- | ----- |
| {COMPONENT_ISSUE_LINK} | {INDICATOR}} | {STATUS} |

<details><summary>Legend</summary>
<p>

| Symbol | Meaning |
| -------- | ---------- |
| :green_circle: | On track with overall release |
| :yellow_circle: | Missed last milestone |
| :red_circle: | Missed multiple milestones |

</p>
</details>

### PR DESCRIPTION
**Pull request by zelinh**
Signed-off-by: Zelin Hao <zelinhao@amazon.com>

### Description
Add JS and CU plugins to 2.0.1 manifest

### Issues Resolved
#2165 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by zelinh** [comment]
Version bump PR were created but not yet merged for both JS and CU. 
https://github.com/opensearch-project/common-utils/pull/186
https://github.com/opensearch-project/job-scheduler/pull/191

File: manifests/2.0.1/opensearch-dashboards-2.0.1.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
```
Comment (gaiksaya) [inline manifests/2.0.1/opensearch-dashboards-2.0.1.yml:1]: Looks like you can remove this change from here and let bot take care of it: https://github.com/opensearch-project/opensearch-build/pull/2170
:)

Comment (gaiksaya) [inline manifests/2.0.1/opensearch-dashboards-2.0.1.yml:1]: Looks like you can remove this change from here and let bot take care of it: https://github.com/opensearch-project/opensearch-build/pull/2170
:)

Comment (zelinh) [reply manifests/2.0.1/opensearch-dashboards-2.0.1.yml]: Thanks. Removed this file.

**Review by gaiksaya — APPROVED** [review]

**Pull request merged**